README.md

```diff
@@ -44,6 +44,12 @@ If you have no idea how to install Flutter and run it locally, check this
 If you have Flutter setup locally, on the project's root folder, install pubspec dependencies by running
 `flutter pub get`.
 
+### Firebase dependencies
+
+Memo has Firebase dependencies that require a setup before the project can be run locally. The original 
+`GoogleServices-Info.plist` (iOS) and `google_service_account` (Android) are not checked in source control, which means 
+that you must provide your own Firebase project google service files to run the app.
```

Comment:
I would also add a link to people that may never had setup a firebase project, so they can start from somewhere:

`If you want to know how to setup your own Firebase project, check out the [FlutterFire docs](https://firebase.flutter.dev/docs/overview/)`